tests/nested/extra-snaps-assertions/task.yaml

```diff
@@ -3,10 +3,17 @@ summary: create ubuntu-core image and execute the suite in a nested qemu instanc
 systems: [ubuntu-16.04-64, ubuntu-16.04-32]
 
 prepare: |
+    # FIXME: until https://github.com/snapcore/snapd/pull/3263 is available from
+    # the archive we need to build snapd from branch so that it can be used by
```

Comment:
Silly question, why do we need to purge snapd? Won't this be replaced with our build snapd in prepare_classic anyway?